cloudinary-core/src/main/java/com/cloudinary/Cloudinary.java

```diff
@@ -153,6 +154,15 @@ public void setUserAgent(String prefix, String version){
         userAgent = prefix+"/"+ version + " ("+USER_AGENT_PREFIX+ " "+VERSION+") " + USER_AGENT_JAVA_VERSION;
     }
 
+    /**
+     * Set the analytics object that will be sent with every URL generation call.
+     * a userAgent is built from `prefix/version (additional data)`
```

Comment:
userAgent? how is it related to the analytics?

Reply:
It's not , it's the function above the `setAnalytics` function

Reply:
Here i just changed the documentation for the `setAnalytics` function.